### ISSUE TITLE
expo optimize error

### ISSUE THREAD

**Issue by XeeshanAnsari** [expo/expo-cli#865]
When i run the expo optimize so i'm getting the following error.

Processing images using sharp-cli failed: sharp exited with non-zero code: 127
Output: vips warning: pngload: no property named `fail'

node: symbol lookup error: /usr/local/lib/node_modules/sharp-cli/node_modules/sharp/build/Release/sharp.node: undefined symbol: _ZNK4vips6VImage7pngsaveEPKcPNS_7VOptionE

**Comment by AdamJNavarro** [comment]
Hey @XeeshanAnsari,

Can you run `expo diagnostics` and share what's printed out here?

Cheers,
Adam

**Comment by jciccio** [comment]
I'm having a similar issue, but each time I try to run expo optimize I get:
```
dest already exists.
Error: dest already exists.
    at pathExists (/Users/jose/.config/yarn/global/node_modules/@expo/xdl/node_modules/fs-extra/lib/move/index.js:46:31)
    at fn.apply.then.r (/Users/jose/.config/yarn/global/node_modules/universalify/index.js:23:46)
```

Already tried installing latest expo-cli

```
 Expo CLI 3.0.2 environment info:
    System:
      OS: macOS Sierra 10.12.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.0 AI-171.4443003
      Xcode: 8.3.3/8E3004b - /usr/bin/xcodebuild
    npmPackages:
      expo: ^33.0.0 => 33.0.7 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8 
      react-navigation: ~2.3.1 => 2.3.1 
```

**Comment by XeeshanAnsari** [comment]
expo diagnostics result:

 Expo CLI 3.0.2 environment info:
    System:
      OS: Linux 4.15 Ubuntu 18.04.2 LTS (Bionic Beaver)
      Shell: 4.4.19 - /bin/bash
    Binaries:
      Node: 12.6.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/bin/yarn
      npm: 6.10.1 - /usr/local/bin/npm
    npmPackages:
      expo: ^33.0.0 => 33.0.7 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8 
      react-navigation: ^3.11.0 => 3.11.0 
    npmGlobalPackages:
      expo-cli: 3.0.2

**Comment by sangampandey** [comment]
Hi XeeshanAnsari,

I had similar issue but mine was related to same file present twice. I have manage to fix it by deleting the duplicate file. Looking into your diagnostic report I will suggest try using Node version 10.16.0 and it might be resolved as I had similar issue and upgrading to Node 10.16.0 fixed the issue for me.

**Comment by NazeerBZ** [comment]
@XeeshanAnsari
I also encounter the same problem and got this resolved.
Run
`sudo npm install --unsafe-perm -g expo-cli`
and then
`expo optimize`

**Comment by EvanBacon** [comment]
This has been fixed in #742
